TM1py/Services/CellService.py

```diff
@@ -357,9 +389,10 @@ def write_value(self, value: Union[str, float], cube_name: str, element_tuple: I
         data = json.dumps(body_as_dict, ensure_ascii=False)
         return self._rest.POST(url=url, data=data, **kwargs)
 
+
     def write(self, cube_name: str, cellset_as_dict: Dict, dimensions: Iterable[str] = None, increment: bool = False,
               deactivate_transaction_log: bool = False, reactivate_transaction_log: bool = False,
-              sandbox_name: str = None, use_ti=False, **kwargs):
+              sandbox_name: str = None, use_ti=False, use_changeset: bool=True, **kwargs):
```

Comment:
I think it would be good to throw a ValueError here if `use_ti` is `True `and `use_changeset` is `True`

Reply:
I will add that